jupyter-mime.el

```diff
@@ -313,6 +313,18 @@ aligns on the current line."
         (setf (image-property image :ascent) 50)
         (force-window-update)))))
 
+(defun jupyter--insert-temp-file-browse-url (data)
+  (let ((file (make-temp-file "emacs-jupyter" nil ".html")))
+    (with-temp-file file (insert data))
+    (browse-url-of-file file)
+    ;; Give the external browser time to open the tmp file before deleting it
+    ;; based on mm-display-external
+    (let ((file file))
```

Comment:
This `let` is also unnecessary, the lambda already captures the `file` variable of the first `let` form.

Reply:
Thanks, I've fixed this now.